### ISSUE TITLE
Preview window is empty

### ISSUE THREAD

**Issue by Yboktaya** [saadq/resumake.io#101]
**Bug description**
The window on the right doesn't output anything. Always white 

**To Reproduce**
1. Go to to the website
2. Click on the desired template
3. Nothing happens 

I tried on different devices with different browsers. same issue, what could be the reason.

**Comment by saadq** [comment]
Yes, I think it's currently unintuitive. There won't be any content shown in the preview window until you put information in to the form fields (using the sidebar on the left). You can press the <kbd>Make</kbd> button whenever you want to generate the PDF.

But yeah, there are plans to make the UX more intuitive in the next version.
